youtube_dl/extractor/pornhub.py

```diff
@@ -165,6 +165,14 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         host = mobj.group('host') or 'pornhub.com'
         video_id = mobj.group('id')
+        cookie_file = self._downloader.params.get('cookiefile')
+
+        if host == 'pornhubpremium.com':
+            if not cookie_file:
+                raise ExtractorError(
+                    'PornHub Premium requires authentication.'
+                    ' You may want to use --cookies.',
```

Comment:
You must provide account credentials/cookies for testing.

Reply:
Sure thing. I emailed you a link to a gist.